src/Rings/mpoly-localizations/methods.jl

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Btw: I introduced this extra file since the recent regression of Revise makes it impossible to have type definitions in the same file you're working on. Eventually, `src/Rings/mpoly-localizations.jl` should be split into several files in this folder, but I didn't want to do that now as it would completely clutter this specific PR.

Reply:
Just pin Revise to 3.4.0 and all should be good?

Reply:
I agree with @fingolfin that we should not change file structures because some other irrelevant package is not working.

Reply:
I see the point. But the original file is already much too big, anyway. So in principle this is a good occasion to start splitting it up. I just didn't want to do it here because one could then not see what this PR is really about. Should I change it back anyway? 

I'm still waiting for the feedback on my actual questions below, though.